api.js

```diff
@@ -29,8 +29,8 @@ module.exports = function (path, remoteApi, _remoteCall) {
     return value
   }
 
-  //add all the api methods to emitter recursively
-  ;(function recurse (obj, api, path) {
+  //add all the api methods to the emitter recursively
+  var recurse = function (obj, api, path) {
```

Comment:
why did you change this? the self evaluating function style is better here because it's explicit that it's not called from anywhere other than inside itself.

Reply:
oh sorry! I see it now.